src/Test/Questar.OneRoster.Test/Mock/Entity.cs

```diff
@@ -1,30 +1,56 @@
-using System.Collections.Generic;
-
 namespace Questar.OneRoster.Test.Mock
 {
     using System;
+    using System.Collections.Generic;
     using static Util;
 
     internal class Entity
     {
         public string FooString { get; set; }
+
         public int BarInt { get; set; }
+
         public DateTime BazDateTime { get; set; }
+
         public Guid QuxGuid { get; set; }
+
         public Count CorgeEnum { get; set; }
 
+        public string[] Subjects { get; set; }
+
+        public Foo Foo { get; set; }
+
+        public Entity Parent { get; set; }
+
+        public Entity[] Children { get; set; }
+
+        internal static string[] GetSubjects()
+        {
+            return new[]
```

Comment:
Lambda syntax.

Reply:
Used Enumerable.Range along with the body expression. Code Cleanup picked up a bit more, too.